woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php

```diff
@@ -494,32 +494,38 @@ public function after_payment_methods_table( $has_methods ) {
 	 *
 	 * @since 5.6.0-dev
 	 *
-	 * @param int $token_id the ID of a core token
+	 * @param int $core_token_id the ID of a core token
 	 * @param \WC_Payment_Token $core_token the core token object
 	 */
-	public function payment_token_deleted( $token_id, $core_token ) {
+	public function payment_token_deleted( $core_token_id, $core_token ) {
 
-		$token = $core_token instanceof \WC_Payment_Token ? $this->get_token_by_id( $core_token->get_token() ) : null;
+		$token_id = null;
 
-		if ( $token instanceof SV_WC_Payment_Gateway_Payment_Token ) {
+		// find out if the core token belongs to one of the gateways from this plugin
```

Comment:
Maybe it would be good to add a comment saying we can't use `get_token_by_id` in this scenario because the FW token was already deleted, for future reference.

Reply:
Good idea. I added another comment in 024f2dc